docs/UserGuide.md

```diff
@@ -174,6 +177,26 @@ Examples:
 * `list` followed by `delete 2` deletes the 2nd attraction in the TrackPad.
 * `find USS` followed by `delete 1` deletes the 1st tourist attraction in the results of the `find` command.
 
+### Editing a tourist attraction: `edit`
```

Comment:
I think there is already another edit?

Reply:
I just realised this too, and this title should have 4 hashtag instead of 3